component/home/home.css

```diff
@@ -0,0 +1,225 @@
+@import url("https://fonts.googleapis.com/css?family=Lato:400,700");
+
+:root{
+    --primaryColor: #1276a5;
+    --secondaryColor:#cf443f;
+    --mainWhite: #fff;
+    --mainBlack: #222;
+    --mainGrey: #ececec;
+    --mainSpacing: 0.1rem;
+    --main-bg:#043044;
+}
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  body {
+    font-family: "Lato", sans-serif;
+  }
+
+  /* navbar starts */
+
+  .docnav-bar {
+	background: var(--main-bg);
```

Comment:
Just a suggestion to indent the code using prettier

Reply:
okay update it